### PR TITLE
enhance lsb_release detection for multi-docker

### PR DESCRIPTION
**Pull request by dalanlan**
Exit for unsupported platform when deploying k8s via multidocker
Related #14494

/cc @mikedanese  @resouer

### CODE REVIEW

**Comment by resouer** [comment]
I noticed a lot of indent problems, please fix that :)

**Comment by dalanlan** [comment]
Nope 
That's what i wanted


Thursday, October 8, 2015, 15:49 +0800 from Harry Zhang  <notifications@github.com>:

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 1]
```diff
@@ -75,6 +75,15 @@ detect_lsb() {
     fi
 
     lsb_dist="$(echo ${lsb_dist} | tr '[:upper:]' '[:lower:]')"
+
+    case "$lsb_dist" in
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: nit: curly braces

File: docs/getting-started-guides/docker-multinode/worker.sh [hunk 2]
```diff
@@ -84,6 +84,15 @@ detect_lsb() {
     fi
 
     lsb_dist="$(echo ${lsb_dist} | tr '[:upper:]' '[:lower:]')"
+
+    case "$lsb_dist" in
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/worker.sh]: nit: curly braces

**Comment by brendandburns** [comment]
LGTM, two nits.

**Comment by mikedanese** [comment]
@dalanlan Thanks!

**Comment by dalanlan** [comment]
@brendandburns  The comments are addressed. PTAL :)

**Comment by dalanlan** [comment]
Any problem with merge robot again? Seems that this patch has been labeled as "ok-to-merge" for quite a while.

**Comment by dalanlan** [comment]
Merge robot must be using a submit "stack" instead of a submit "queue" :-/

**Comment by mikedanese** [comment]
@dalanlan ok-to-merge does not mean this PR has been lgtm'd, it just means this PR can be accepted from a person who's not on the whitelist. PR's that are marked ok-to-merge still need an lgtm. It's really confusing. Please submit a PR to add your name to the whitelist here:

https://github.com/kubernetes/contrib/blob/master/mungegithub/whitelist.txt

**Comment by dalanlan** [comment]
Thank you for much for the clarification:)

**Pull request merged**